definitions/private-law/json/CaseField/CaseFileView/CaseField.json

```diff
@@ -541,12 +551,22 @@
     "LiveFrom": "24/11/2022",
     "CaseTypeID": "PRLAPPS",
     "ID": "homeOfficeDWPResponses",
-    "Label": "Home office/ DWP responses",
+    "Label": "Home Office/ DWP responses",
```

Comment:
Office or office?

Reply:
Hi @shashisk97, as requested by @khurshedk , I am commenting here,
As part of https://tools.hmcts.net/jira/browse/PRL-4545 'o' has been changed to 'O' as suggested in https://tools.hmcts.net/confluence/display/PL/Case+File+View+-+Document+Category+V0.4